src/Network/Network.Test/ScenarioTests/AzureFirewallPolicyTests.ps1

```diff
@@ -1492,7 +1492,7 @@ function Test-AzureFirewallPolicyPrivateRangeCRUD {
     $location = "westus2"
     $vnetName = Get-ResourceName
     $privateRange2 = @("IANAPrivateRanges", "0.0.0.0/0", "66.92.0.0/16")
-    $privateRange1 = @("3.3.0.0/24", "98.0.0.0/8")
+    $privateRange1 = @("3.3.0.0/24", "98.0.0.0/8","10.227.16.0/20")
```

Comment:
The value of $privateRange1 changes, these two test cases need to re-record.

Reply:
@BethanyZhou updated session records.